### ISSUE TITLE
State icon depending on state value

### ISSUE THREAD

**Issue by braindead1** [Zefau/ioBroker.jarvis#61]
If I remember correctly it was possible to define a state icon depending on the state value in a previous version. I believe this was done in the JSON.

It would be great if this could be implemented in the UI. Instead of just offering the possibility to enter an icon name, it should be possible to enter JSON similar to state style and display variant.

    {
        "true": "account",
        "false": "account-off"
    }

![image](https://user-images.githubusercontent.com/2136249/92411198-21303180-f147-11ea-9b8b-e03091778024.png)

If possible, it would be great to have the following for numeric values:

    {
        "<20": "battery-20",
        ">=20": "battery"
    }

**Comment by Zefau** [comment]
The input was not parsed as json. This has been fixed with `1.0.0-beta.17`. Using `<`, `<=` and `>`, `>=` as well as `!=` should already be possible (and now working as well).

**Comment by braindead1** [comment]
I have installed `v1.0.0-beta.17`.

Entering JSON for boolean states works as expected.

It is not possible to just enter the name of an icon as there is directly the message that the JSON is not valid.

![image](https://user-images.githubusercontent.com/2136249/92476034-a1ee3c80-f1de-11ea-91ca-f35385246713.png)

I tried to use the following JSON:

    {
        "<=40": "battery-40",
        ">40": "battery"
    }

For the `battery` state it has no effect and the default icon is used. For a slef-created state no icon is displayed.
